ps/ps.map/do_scalebar.c

```diff
@@ -5,6 +5,7 @@
 #include <grass/glocale.h>
 #include "local_proto.h"
 #include "distance.h"
+#include <grass/gis.h>
```

Comment:
Move this include line before `#include <grass/glocale.h>`. 

Better to group (and alphabetically sort) the different include "levels" (system, grass lib and local).

(There is no recommended order of the include levels within the GRASS GIS project, so for the time being we just keep the existing order in each file.)

Reply:
FWIW: I darkly remember that we reordered the includes to exactly this order long time ago during a major C code refactoring activity (it degraded a bit over time). Maybe worth adding a note here?

https://github.com/OSGeo/grass/blob/main/doc/development/style_guide.md#c-and-c

Reply:
Ahh, I see. Presently, there is a mix of include "level"-order though. I know I noticed this when I did some testing with the ClangFormat [SortIncludes](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#sortincludes) setting, but ended up to set it to `Never`. The levels need to be separated by new line for ClangFormat not apply sort across the board, and this means some (!) manual work.
I agree this should be added to the guide.

Reply:
Found it!  https://trac.osgeo.org/grass/wiki/Submitting/C#OtherHeaders

Reply:
> Found it! https://trac.osgeo.org/grass/wiki/Submitting/C#OtherHeaders

I must have missed this. I will add relevant parts unless somebody is working on it already.

Reply:
> > Found it! https://trac.osgeo.org/grass/wiki/Submitting/C#OtherHeaders
> 
> I must have missed this. I will add relevant parts unless somebody is working on it already.

Please go ahead, I haven't (yet) started.

Reply:
> Move this include line before `#include <grass/glocale.h>`.
> 
> Better to group (and alphabetically sort) the different include "levels" (system, grass lib and local).
> 
> (There is no recommended order of the include levels within the GRASS GIS project, so for the time being we just keep the existing order in each file.)

Done